forum/qa-include/app/upload.php

```diff
@@ -39,13 +39,13 @@ function qa_get_max_upload_size()
 
 		switch (strtolower(substr($minphp, -1))) {
 			case 'g':
-				$minphp *= 1024;
+				$minphp = (int) $minphp * 1024;

```

Comment:
ten kawałek kodu rzucał notice wcześniej - chodziło o to, że próbowało się zrobić takie coś:
```
$minphp = '2m';
$minphp *= 1024;
```
To skutkowało tym, że PHP próbował zrobić:
2M * 1024 <- co w wersji 5 nie robiło problemu bo jest rzutowanie niejawne, ale w nowej wersji już rzuca błędami bo próbujemy mnożyć string przez int.